warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__lead_time.sql

```diff
@@ -53,7 +53,7 @@ trips_expanded AS (
       LEFT JOIN dim_stops t3
         ON t3.stop_id = t2.stop_id
        AND t3.feed_key = t2.feed_key
-     GROUP BY 1,2,3
+     GROUP BY 1, 2, 3 -- noqa
```

Comment:
what is "-- noqa"?

Reply:
It's to suppress [SQLFluff](https://docs.sqlfluff.com/en/stable/) warnings on this line. @atvaccaro said he had to turn off SQLFluff in CI so it hasn't been running. But thanks for flagging, I think this suppression is probably too intense and I should be more specific to turn off specific rules